### ISSUE TITLE
ModifyWorld is being weird [HELP!!!]

### ISSUE THREAD

**Issue by AugustZellmer** [PEXPlugins/PermissionsEx#331]
Please help. This is really bugging me!

I added a new world to my server. I was able to edit it. 

Then I wasn't allowed to use the permissions of a different plug-in in that world. This started happening right after I used a command for the first time.

I went in the PEX permissions.yml and added 
world:
-PF:
--permissions:
---modifyworld.*
---trp.*

I went back and I still wasn't allowed to do either of those things.

I undid everything I did to the PEX to see if I messed something up. It still wont' let me either of those things.

Then I realized the first plug-in had nothing to do with it.
Basically, ModifyWorld is not allowing me to edit my world or use plug-ins that add blocks.
I went on another world and I could edit blocks, but still not use plug-ins that add blocks.
This all starts the first time you use a command that tries to add blocks.

This is ruining everything. Please help me.
